vonage/datadog_checks/vonage/data/conf.yaml.example

```diff
@@ -0,0 +1,392 @@
+## All options defined here are available to all instances.
```

Comment:
I don't think you need anything under the `/datadog_checks` folder, since this is not an agent integration.